Jenkinsfile

```
@@ -16,6 +16,8 @@ node ('!(ci-agent-4)') {
     stage('Installing Packages') {
       sh("rm -rf ./venv")
       sh("virtualenv --no-site-packages ./venv")
+      sh("curl https://bootstrap.pypa.io/2.7/get-pip.py -o get-pip.py")
+      sh("bash -c 'source venv/bin/activate && python get-pip.py pip==20.3.4'")
```

```suggestion
      sh("bash -c 'venv/bin/python get-pip.py pip==20.3.4'")
```

I think this would avoid the need to run `source`.